.github/workflows/build-release.yml

```diff
@@ -18,17 +18,17 @@ jobs:
       - name: Generate index.php
         run: ~/.composer/vendor/bin/autoindex
       - name: Create & upload artifact
-        uses: actions/upload-artifact@v1
+        uses: actions/upload-artifact@v3
         with:
           name: ${{ github.event.repository.name }}
-          path: ../
+          path: /home/runner/work/${{ github.event.repository.name }}
```

Comment:
is it necessary to change this? do you have any source confirming this change?

Reply:
@leemyongpakvn

Reply:
@kpodemski upload-artifact v3 does not allow ../ as path, I will change back to ../ then you will can the error.
The first confirmation is at https://github.com/PrestaShop/blockwishlist/pull/216/files
The second confirmation is at https://github.com/PrestaShop/pagesnotfound/pull/29/files :)